### PR TITLE
Start charts on contest stats page at 0; Fixes #1231

### LINKED ISSUE #1231: Empty graphs on contest stat pages should start at 0.0
Current, the charts on the statistics pages for contests start at -1.0 when they are empty, which doesn't make sense (how can you have a -1.0 AC rate?)

These should be changed to start at 0.0 instead.

### PR DESCRIPTION
**Pull request by Carson-Tang**

### CODE REVIEW

File: templates/stats/media-js.html [hunk 1]
```diff
@@ -57,6 +64,9 @@
                 scales: {
                     xAxes: [{
                         stacked: true,
+                        ticks: {
+                            min: 0,
+                        }
```
Comment (Xyene) [inline templates/stats/media-js.html]: comma

File: templates/stats/media-js.html [hunk 2]
```diff
@@ -38,6 +38,13 @@
                         },
                     },
                 },
+                scales: {
+                    xAxes: [{
+                        ticks: {
+                            min: 0,
+                        }
```
Comment (Xyene) [inline templates/stats/media-js.html]: comma

**Comment by Carson-Tang** [comment]
image

File: templates/stats/media-js.html (hunk 2 above)
Comment (Xyene) [reply templates/stats/media-js.html]: wait it's not resolved

Comment (Xyene) [reply templates/stats/media-js.html]: I meant add one, not remove another one

**Comment by Xyene** [comment]
Thanks!

**Pull request merged**